testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/microprofile/opentracing/EarOpenTracingTestCase.java

```diff
@@ -75,13 +75,11 @@
         return ear;
     }
 
-    @Ignore("Unignore when https://issues.jboss.org/browse/WFWIP-104") // TODO
```

Comment:
The import needs to be removed as this is causing a build failure.

Reply:
Import removed